config/checkstyle-non-main-files-suppressions.xml

```diff
@@ -253,9 +253,6 @@
   <!-- until https://github.com/checkstyle/checkstyle/issues/13839 -->
   <suppress id="propertiesMacroMustExist"
          files="src[\\/]xdocs[\\/]checks[\\/]annotation[\\/]suppresswarningsholder.xml.template"/>
-  <!-- until https://github.com/checkstyle/checkstyle/issues/13893 -->
```

Comment:
i have already included it as a part of https://github.com/checkstyle/checkstyle/pull/13971. can we wait for it to be merged or i will have to rebase?

Reply:
Better to fix failing CI quicker.

Reply:
thank you @nrmancuso , marking this conversation as resolved.